src/app/pages/TopicPage/Curation/index.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { arrayOf, oneOf, shape, string, number } from 'prop-types';
 import pathOr from 'ramda/src/pathOr';
+import VisuallyHiddenText from '#psammead/psammead-visually-hidden-text/src';
```

Comment:
Should we be importing from psammead still?

Reply:
We merged Psammead into Simorgh. This is import from the psammead dir within Simorgh